app/controllers/api/v1/salons_controller.rb

```diff
@@ -0,0 +1,58 @@
+class Api::V1::SalonsController < ApplicationController
+  def index
+    @salons = Salon.all
+    render json: salons
+  end
+
+  def show
+    @salon = Salon.find_by(id: params[:id])
+    if salon
+      render json: @salon
+    else
+      render json: { error: 'Error creating review.' }
+    end
+  end
+
+  def create
+    @salon = Salon.new(
```

Comment:
Probably you can write it as Salon.new(salon_params), I'm not sure, but it should work. check it please